TDataFrame.hxx

```diff
@@ -856,182 +1070,172 @@ public:
                }
             }
 
-            BuildAllReaderValues(r, slot); // TODO: if TTreeProcessor reuses the same TTreeReader object there is no need to re-build the TTreeReaderValues
+            BuildAllReaderValues(r, slot); // TODO: if TTreeProcessor reuses the same TTreeReader object there is no
+                                           // need to re-build the TTreeReaderValues
```

Comment:
it turns out `TTreeProcessor` does *not* reuse the same `TTreeReader` object, this line is fine as it is